app/src/models/answersModel.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+const ObjectId = Schema.ObjectId;
+
+var QuestionResponse = new Schema({
+    name: {type: String, required: true, trim: true},
```

Comment:
Same than below so maybe we can generate it in the POST request instead of sending in the body params.